### PR TITLE
IDE-2195 fix no display name issue

### PR DESCRIPTION
**Pull request by AndyWu2015**

### CODE REVIEW

File: com.liferay.blade.cli/templates/standalone/mvcportlet/src/main/java/_package_path_/_CLASSNAME_.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
 	property = {
 		"com.liferay.portlet.display-category=category.sample",
 		"com.liferay.portlet.instanceable=true",
+		"javax.portlet.display-name=_NAME_ JSP Portlet",
```
Comment (gamerson) [inline com.liferay.blade.cli/templates/standalone/mvcportlet/src/main/java/_package_path_/_CLASSNAME_.java]: Lets just call this one ```_NAME_ Portlet``` as well.

ALso @AndyWu2015 can you add a test case for making sure that this substitution is working as expected?  Thanks!

**Comment by gamerson** [comment]
Closing see comments, thanks!

File: com.liferay.blade.cli/templates/standalone/mvcportlet/src/main/java/_package_path_/_CLASSNAME_.java (hunk 1 above)
Comment (AndyWu2015) [inline com.liferay.blade.cli/templates/standalone/mvcportlet/src/main/java/_package_path_/_CLASSNAME_.java]: I've tested it locally and worked well , you mean that we need add a case : create a mvnportlet and deploy it into portal and check display name in portal or in gogo shell ?
I think it is hard to do this .

Comment (gamerson) [inline com.liferay.blade.cli/templates/standalone/mvcportlet/src/main/java/_package_path_/_CLASSNAME_.java]: no no, just make a test case that searches in the output portlet.java file to make sure the display name is as expected.

Comment (AndyWu2015) [inline com.liferay.blade.cli/templates/standalone/mvcportlet/src/main/java/_package_path_/_CLASSNAME_.java]: ok

Comment (AndyWu2015) [inline com.liferay.blade.cli/templates/standalone/mvcportlet/src/main/java/_package_path_/_CLASSNAME_.java]: done

**Pull request closed without merging**